lib/api_remote.ml

```diff
@@ -335,29 +322,22 @@ module Buildkite : Api.Buildkite = struct
     | Some access_token ->
       let headers = bearer_token_header access_token :: Option.default [] headers in
       let url = sprintf "https://api.buildkite.com/v2/%s" path in
-      (match%lwt buildkite_api_request ?body ~headers meth url read with
-      | Ok res -> Lwt.return @@ Ok res
-      | Error e -> Lwt.return @@ fmt_error "%s: failure : %s" name e)
+      buildkite_api_request ?body ~headers meth url read
+      |> Lwt_result.map_error (fun e -> sprintf "%s: failure : %s" name e)
```

Comment:
```suggestion
      |> Lwt_result.map_error (sprintf "%s: failure : %s" name)
```

Reply:
I am not sure with that one. having the list of of parameters at the end of the format string improves readability in my view.